fineract-provider/src/main/java/org/apache/fineract/portfolio/address/domain/Address.java

```diff
@@ -0,0 +1,348 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.portfolio.address.domain;
+
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import org.apache.fineract.infrastructure.codes.domain.CodeValue;
+import org.apache.fineract.infrastructure.core.api.JsonCommand;
+import org.apache.fineract.portfolio.client.domain.ClientAddress;
+import org.springframework.data.jpa.domain.AbstractPersistable;
+
+import com.google.gson.JsonObject;
+
+@Entity
+@Table(name = "m_address")
+public class Address extends AbstractPersistable<Long> {
+
+	/*@OneToMany(mappedBy = "address", cascade = CascadeType.ALL)
+	private List<ClientAddress> clientaddress = new ArrayList<>();*/
+	
+	@OneToMany(mappedBy = "address", cascade = CascadeType.ALL)
+	private Set<ClientAddress> clientaddress;
+
+	private String street;
+
+	private String address_line_1;
```

Comment:
Give proper names to all variables in entity classes. You can refer other entity classes